homeassistant/components/sensor/rmvtransport.py

```diff
@@ -0,0 +1,209 @@
+"""
+Support for real-time departure information for Rhein-Main public transport.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.rmvdeparture/
+"""
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_NAME, ATTR_ATTRIBUTION, STATE_UNKNOWN
+    )
+
+REQUIREMENTS = ['PyRMVtransport==0.0.5']
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_NEXT_DEPARTURE = 'nextdeparture'
+
+CONF_STATION = 'station'
+CONF_DESTINATIONS = 'destinations'
+CONF_DIRECTIONS = 'directions'
+CONF_LINES = 'lines'
+CONF_PRODUCTS = 'products'
```

Comment:
`products` is weird for me, maybe `vehicle`?

Reply:
’product’ is the wording taken from the providers public API and it is their product so it made sense to me to keep the wording the same. Maybe I should add some comments.